packages/vm/src/middleware/deterministic.rs

```diff
@@ -138,11 +138,365 @@ impl FunctionMiddleware for FunctionDeterministic {
             | Operator::I64Extend8S
             | Operator::I64Extend16S
             | Operator::I64ExtendI32S
-            | Operator::I64ExtendI32U => {
+            | Operator::I64Extend32S
+            | Operator::I64ExtendI32U
+            | Operator::RefNull { .. }
+            | Operator::RefIsNull
+            | Operator::RefFunc { .. }
+            | Operator::ReturnCall { .. }
+            | Operator::ReturnCallIndirect { .. }
+            | Operator::TypedSelect { .. }
+            | Operator::MemoryInit { .. }
+            | Operator::DataDrop { .. }
+            | Operator::MemoryCopy { .. }
+            | Operator::MemoryFill { .. }
+            | Operator::TableInit { .. }
+            | Operator::ElemDrop { .. }
+            | Operator::TableCopy { .. }
+            | Operator::TableFill { .. }
```

Comment:
Those belong to https://github.com/WebAssembly/bulk-memory-operations/blob/master/proposals/bulk-memory-operations/Overview.md and should be rejected with a different error message

Reply:
From that Markdown doc it looks like `MemoryInit` too?

Reply:
Yes, thanks for double checking those. I'm not very familiar with what those extensions do and which operations they add.